tests/cases/fourslash/completionForComputedStringProperties.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference path="fourslash.ts" />
+
+//// const p2 = "p2";
+//// interface A {
+////     ["p1"]: string;
+////     [p2]: string;
+//// }
+//// declare const a: A;
+//// a./*1*/ ;
+//// a["/*2*/ ;
+
+verify.completions({ marker: "1", exact: "p1" });

```

Comment:
Shouldn't `p2` show up here?

Reply:
@DanielRosenwasser Maybe you can clarify and point me out how to trigger in `verify.completions` list for `.` and `[]` which will contain all available properties. For instance server returns both properties, however, behave differently

![1](https://user-images.githubusercontent.com/509265/57545583-74338200-7363-11e9-863f-182c5c3cd95a.png)

when choosing dynamic declared property dot will be replaced by `[]`. Maybe for that reason, the first version doesn't contain dynamic declared property.

Reply:
@DanielRosenwasser what do you think?

Reply:
I think you need to pass user preferences with `includeInsertTextCompletions` set to `true`. Eg check `tests/cases/fourslash/completionsThisType.ts `

Reply:
@sheetalkamat thanks ;)